src/audio/drc/drc_math_generic.c

```diff
@@ -127,11 +127,11 @@ inline int32_t drc_asin_fixed(int32_t x)
 #define qch 26
 	/* Coefficients obtained from:
 	 * If x <= 1/sqrt(2), then
-	 *   fpminimax(asin(x), [|1,3,5,7|], [|SG...|], [-1e-30;1/sqrt(2)], absolute)
-	 *   max err ~= 1.89936e-5
+	 *	  fpminimax(asin(x), [|1,3,5,7|], [|SG...|], [-1e-30;1/sqrt(2)], absolute)
+	 *	  max err ~= 1.89936e-5
 	 * Else then
-	 *   fpminimax(asin(x), [|1,3,5,7|], [|SG...|], [1/sqrt(2);1], absolute)
-	 *   max err ~= 3.085226e-2
+	 *	  fpminimax(asin(x), [|1,3,5,7|], [|SG...|], [1/sqrt(2);1], absolute)
+	 *	  max err ~= 3.085226e-2
 	 */
 	const int32_t TWO_OVER_PI = Q_CONVERT_FLOAT(0.63661977236758134f, qcl); /* 2/pi */
```

Comment:
What is "qcl"? It should be a macro or literal, otherwise Q_CONVERT_FLOAT() produces float library C code and increases a lot code size.

Also I'd not use "f" for decimal number literals since by default without a type letter it would be double precision with better precision.